templates/values.yaml.tpl

```diff
@@ -59,4 +59,12 @@ kas:
 
   redis:
     auth:
-      password: {{ .Values.kasRedis }}
\ No newline at end of file
+      password: {{ .Values.kasRedis }}
+
+dashboard:
+  api:
+    containers:
+      args:
+        - --apiserver-host=https://{{ .Values.kasDns }}/k8s-proxy
```

Comment:
We should also add this to the `templates/values.yaml.liquid`.  I think this is used by plural up the other by `plural cd control-plane` which generates helm values for byok installs

Reply:
It is already there. I have added it to the initial PR that introduced dashboard dependency. Just forgot to update this one too.